packages/table/src/common/loading.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+export interface ILoadable {
+    /**
+     * Show an animated loading animation.
+     * @default false
+     */
+    isLoading?: boolean;
```

Comment:
i think i'd prefer to call it simply `loading` [like we're doing with buttons](https://github.com/palantir/blueprint/pull/349/files#diff-76186512bf8bfa4ccfa05cb3d9372687R30)